Adding AWS account id and GCP project number and name

veeruns opened the pull request:
Athenz domains contain cloud provider account data. adding gcp and was account data to the data source all_domain_details.

mendi160 reviewed (commented):
LGTM

mendi160 reviewed (commented):
Please make sure to update the doc file as well 
docs/data-sources/athenz_all_domain_details.md

veeruns commented:
> Please make sure to update the doc file as well docs/data-sources/athenz_all_domain_details.md

updated doc, and added azure subscription to the exported attribute.
